CMSIS/NN/Tests/UnitTest/TestCases/test_arm_depthwise_conv_s8/test_arm_depthwise_conv_s8.c

```diff
@@ -558,4 +579,67 @@ void depthwise_null_bias_1_arm_depthwise_conv_s8(void)
     free(ctx.buf);
     TEST_ASSERT_EQUAL(expected, result);
     TEST_ASSERT_TRUE(validate(output, depthwise_null_bias_1_output_ref, DEPTHWISE_NULL_BIAS_1_DST_SIZE));
-}
\ No newline at end of file
+}
+
+void depthwise_dilation_arm_depthwise_conv_s8(void)
+{
+    const arm_status expected = ARM_MATH_SUCCESS;
+    q7_t output[DEPTHWISE_DILATION_DST_SIZE] = {0};
+
+    cmsis_nn_context ctx;
+    cmsis_nn_dw_conv_params dw_conv_params;
+    cmsis_nn_per_channel_quant_params quant_params;
+    cmsis_nn_dims input_dims;
+    cmsis_nn_dims filter_dims;
+    cmsis_nn_dims bias_dims;
+    cmsis_nn_dims output_dims;
+
+    const q31_t *bias_data = get_bias_address(depthwise_dilation_biases, DEPTHWISE_DILATION_OUT_CH);
+    const q7_t *kernel_data = depthwise_dilation_weights;
+    const q7_t *input_data = depthwise_dilation_input;
+
+    input_dims.n = DEPTHWISE_DILATION_INPUT_BATCHES;
+    input_dims.w = DEPTHWISE_DILATION_INPUT_W;
+    input_dims.h = DEPTHWISE_DILATION_INPUT_H;
+    input_dims.c = DEPTHWISE_DILATION_IN_CH;
+    filter_dims.w = DEPTHWISE_DILATION_FILTER_X;
+    filter_dims.h = DEPTHWISE_DILATION_FILTER_Y;
+    output_dims.w = DEPTHWISE_DILATION_OUTPUT_W;
+    output_dims.h = DEPTHWISE_DILATION_OUTPUT_H;
+    output_dims.c = DEPTHWISE_DILATION_OUT_CH;
+
+    dw_conv_params.padding.w = DEPTHWISE_DILATION_PAD_X;
+    dw_conv_params.padding.h = DEPTHWISE_DILATION_PAD_Y;
+    dw_conv_params.stride.w = DEPTHWISE_DILATION_STRIDE_X;
+    dw_conv_params.stride.h = DEPTHWISE_DILATION_STRIDE_Y;
+    dw_conv_params.dilation.w = DEPTHWISE_DILATION_DILATION_X;
+    dw_conv_params.dilation.h = DEPTHWISE_DILATION_DILATION_Y;
+
+    dw_conv_params.ch_mult = DEPTHWISE_DILATION_CH_MULT;
+
+    dw_conv_params.input_offset = DEPTHWISE_DILATION_INPUT_OFFSET;
+    dw_conv_params.output_offset = DEPTHWISE_DILATION_OUTPUT_OFFSET;
+    dw_conv_params.activation.min = DEPTHWISE_DILATION_OUT_ACTIVATION_MIN;
+    dw_conv_params.activation.max = DEPTHWISE_DILATION_OUT_ACTIVATION_MAX;
+    quant_params.multiplier = (int32_t *)depthwise_dilation_output_mult;
+    quant_params.shift = (int32_t *)depthwise_dilation_output_shift;
+
+    ctx.buf = NULL;
+    ctx.size = 0;
+
+    arm_status result = arm_depthwise_conv_s8(&ctx,
```

Comment:
Lets call the wrapper here so the entire chain is verified.

Reply:
Done